### ISSUE TITLE
iOS App Store blocked

### ISSUE THREAD

**Issue by RouterCFW** [EnergizedProtection/block#917]
<h1 align="center">Energized Protection - block ⚡</h1>
<div align="center">
  <strong>Let's make an annoyance free, better open internet, altogether!</strong>
</div>

<h2 align="center">Issue Submit Form</h2>
<div align="center">
  Provide the following info properly, which will help us to resolve your issue quickly.
</div>

### Issue(s):
Type `x` in between `[ ]` and make sure there isn't any space between brackets. Example; for Your Selected Issue(s), type like this - `[x]` 
You can select more than one category of issues if you need to!

- [ ] Whitelist
- [ ] Blacklist
- [x] App Broken
- [ ] Website Broken
- [ ] Request
- [ ] Other Issue

<hr>

### Pack(s):
Write the name of the pack(s) you are using.

- Pack(s) Name: Energized spark

<hr>

### Extension(s):
Also name the extension(s) if you are using any. If you don't then leave it blank.

- Extension(s) Name: 

<hr>

### Domain(s):
If you are submitting this issue for whitelist/blacklist issue, send us the domain(s) for whitelisting/blacklisting here. Kindly use the **Code Tag** to prevent tracking.

- Domain(s):

```
amp-api-edge.apps.apple.com

```

<hr>

### Your Config:
Just to ensure there is no issues or conflicts with other app/software/magisk module/extension/source list. 
Make sure you are running Energized Protection Service only.

- Client: Next Dns

- Version: Latest 

<hr>

### Detail:
Write us a lil bit more about your issue or query. You can attach any screen shot or log of the issue or advert, this will help to highlight it.

- Your Issue Detail: 

When domain is blocked App Store does not work 

<div align="center">
  <h2>Thank you for making Energized Protection great, with your kind help!</h2>
  <sub>A project by <a href="https://nayemador.com" target="_blank">Ador</a> with ❤<pub>
</div>

**Comment by RouterCFW** [comment]
> Not sure which upstream list affecting this, but it looks very aggressive, not like the thing that the most lightweight blocking pack should do 😆

Yeah but hopefully the maintainer of Energized comes back soon since he just emailed me back 
[https://cdn.starfiles.co/files/ff165cb50ed64a9b4467fd6feb93b16050bc66da1a3378e0ccddbfe4890239d6/E626A500-4452-4A33-82DA-6691FDB8E59D.jpeg](url)

**Comment by ZPrimed** [comment]
Can definitely confirm that `amp-api-edge.apps.apple.com` is the root cause here; I whitelisted *.apps.apple.com and the store started working again for me.

**Comment by PeterDaveHello** [comment]
To [EnergizedProtection/block](https://github.com/EnergizedProtection/block), I found that the upstream the affects this issue actually came from here:

https://github.com/AdroitAdorKhan/EnergizedProtection/commit/7f733fa3febc262bccfc277e38a9861af097bf81#diff-b4bc6903e0b71bb158b54d9b18d5a852b24415820d37e85051abc1d08b30f9e4R6714

```diff
@@ -6710,6 +6711,7 @@
 0.0.0.0 amorluv.com
 0.0.0.0 amountdonutproxy.com
 0.0.0.0 amountinsulation.com
+0.0.0.0 amp-api-edge.apps.apple.com
 0.0.0.0 amp-error-reporting.appspot.com
 0.0.0.0 amp-service.addthis.com
 0.0.0.0 amp-test.adzerk.net
```

Just I don't know the upstream sources of [AdroitAdorKhan/EnergizedProtection](https://github.com/AdroitAdorKhan/EnergizedProtection) project, there's no source mentioned in the project, but it looks like to be updated automatically, so I imagine there are other sources of it. Also, it doesn't look like that there's the place that someone will send a pull request to add or remove domain, so the issue will need to wait for its maintainer to clarify it.
